static/js/ducks/gcnEvent.js

```diff
@@ -27,4 +44,25 @@ const reducer = (state = null, action) => {
   }
 };
 
+export const submitObservationPlanRequest = (params) => {
+  const { instrument_name, ...paramsToSubmit } = params;
+  return API.POST(
+    "/api/observation_plan",
+    SUBMIT_OBSERVATION_PLAN_REQUEST,
+    paramsToSubmit
+  );
+};
+
+export const editObservationPlanRequest = (params, requestID) => {
+  const { instrument_name, ...paramsToSubmit } = params;
+  return API.PUT(
```

Comment:
I'm not sure there is a PUT method in that handler. It may even be better to add a PATCH instead, as it looks like we are going in the direction of replacing all PUT with PATCH.